### PR TITLE
Changed caption of my slide

### PR DESCRIPTION
**Pull request by andrew-scoppa**

### CODE REVIEW

**Review by andrew-scoppa — COMMENTED** [review]
I think this is great!

**Review by muzotabag — APPROVED** [review]
Carpe Diem

**Review by bfmonroe — APPROVED** [review]
Well done

**Review by WallyHartshorn — APPROVED** [review]
Nifty.

**Review by Chembros123 — APPROVED** [review]

File: _slides/02-andrew-scoppa.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
```
Comment (timp70291) [inline _slides/02-andrew-scoppa.md:5]: Reviewed by Tim

**Review by timp70291 — APPROVED** [review]

**Pull request merged**